lib/dl_file_uploader_lib/pyproject.toml

```diff
@@ -21,6 +21,7 @@ dl-utils = {path = "../dl_utils"}
 dl-constants = {path = "../dl_constants"}
 dl-connector-bundle-chs3 = {path = "../dl_connector_bundle_chs3"}
 dl-configs = {path = "../dl_configs"}
+dl-api-lib = {path = "../dl_api_lib"}
```

Comment:
Surely this import can be avoided? We don't want dl-api-lib as a dependency to all file-uploader packages